### PR TITLE
Avalonia now targets .NET 5

### PR DESCRIPTION
**Pull request by bclehmann**
**New Contributors:**
Please review [CONTRIBUTING.md](https://github.com/swharden/ScottPlot/blob/master/CONTRIBUTING.md)

**Hacktoberfest Participants:**
Check-out the [Hacktoberfest 2021](https://github.com/ScottPlot/ScottPlot/issues/1274) page

**Purpose:**
#1306

### CODE REVIEW

**Review by swharden — COMMENTED** [review]
I'll wait until #1310 is merged to avoid conflicts, then I'll merge this in 👍

**Review by swharden — APPROVED** [review]

**Comment by bclehmann** [comment]
It looks like this was never added to the changelog, I don't know if you missed it or thought it wasn't important enough.

I'll make a PR and if you feel it's not necessary then we can close it.

**Pull request merged**
